case-lib/lib.sh

```diff
@@ -41,7 +41,9 @@ fi
 func_lib_setup_kernel_last_line()
 {
     # shellcheck disable=SC2034 # external script will use it
-    KERNEL_LAST_LINE=$(wc -l /var/log/kern.log|awk '{print $1;}')
+    KERNEL_LAST_LINE=$(journalctl --dmesg --no-pager -n 1 -o short-iso-precise|awk '/kernel/ {print $1;}')
```

Comment:
On my system this prints the boot time (uptime -s) which doesn't seem to match what was before.

Should this simply be `KERNEL_LAST_LINE=$(date)`?

I like that you're trying to save a lot of changed lines by keeping the KERNEL_LAST_LINE name the same, however there should be a big fat warning comment saying "This is not a line anymore"

Reply:
I rename the whole thing as KERNEL_LAST_TIMESTAMP in PR #354